PPI/manual_work/May_2023/PPI_concepts_correction_May2023.sql

```diff
@@ -0,0 +1,65 @@
+--concept_manual_backup
+DO
+$body$
+    DECLARE
+        update text;
+    BEGIN
+        SELECT TO_CHAR(CURRENT_DATE, 'YYYY_MM_DD')
+        INTO update;
+        EXECUTE format('create table %I as select * from concept_manual',
+                       'concept_manual_backup_' || update);
+    END
+$body$;
+
+
+
+-- concept_relationship_manual backup
+DO
+$body$
+    DECLARE
+        update text;
+    BEGIN
+        SELECT TO_CHAR(CURRENT_DATE, 'YYYY_MM_DD')
+        INTO update;
+        EXECUTE format('create table %I as select * from concept_relationship_manual',
+                       'concept_relationship_manual_backup_' || update);
+    END
+$body$;
+
+
+
+-- Update 3 concepts and set different concept_class_id for 2 concepts
+
+--Deprecating existing mappings and making two question concepts standard
```

Comment:
We can easily insert the content directly to the manual tables and publish the respective csvs

Reply:
And we need to organize the manual tables as the full set